src/app/screens/coinDashboard/transactionsHistoryList.tsx

```diff
@@ -100,8 +100,10 @@ const groupBtcTxsByDate = (
     if (transaction.txStatus === 'pending') {
       pendingTransactions.push(transaction);
     } else {
-      processedTransactions[txDate] = [transaction] || [];
-      processedTransactions[txDate].push(transaction);
+      !processedTransactions[txDate]
+        ? (processedTransactions[txDate] = [transaction] || [])
```

Comment:
Even when the `transaction` is `undefined` the fallback value won't be used here as the first value will still be an array, so should we remove the `|| []` here?

Reply:
yes makes sense I updated it. 👍